src/routes/docs/tutorials/pagination/step-1/+page.markdoc

```diff
@@ -0,0 +1,30 @@
+---
+layout: tutorial
+title: Build a todos app with react
+description: Learn about using pagination in a react app using appwrite backend.
+step: 1
+difficulty: beginner
+readtime: 10
+---
+
+**Todos App**: An app to track all your todos that you'll need to get done. In this tutorial, you will build the todos app with appwrite and react.
```

Comment:
Focus on the end goal, title and this part should focus on general pagination with Appwrite + react, not the app.